doc/source/examples/rpt/photon-count-calculation-in-a-cylindrical-vessel/photon-count-calculation-in-a-cylindrical-vessel.rst

```diff
@@ -0,0 +1,325 @@
+
+The Case II figure shows the evolution of the photon count in absence of attenuation due to the medium found inside the vessel and the vessel's wall, and in absence of variation of the interaction between the emitted :math:`\gamma`-ray and the detector. By setting :math:`\mu_r = 0`, we set :math:`f_a(\alpha_j, \theta_j) = 1`. As a consequence, the count becomes independent of the path of the photon inside the vessel. In a similar manner, by setting :math:`\mu_d = 1e9`, we make :math:`f_d(\alpha_j, \theta_j)` tend to :math:`1`. Consequently, the path traveled by the photon in the detector doesn't affect the efficiency anymore. Only the weighting factors :math:`\omega(\alpha)` and :math:`\omega(\theta)` have an influence on the calculated efficiency and photon count (:math:`\xi_i \approx \omega(\alpha) \omega(\theta)`). Therefore, the Case II figure gives us an idea of how the photon count evolves according to the particle's position respective to the detector's position disregarding the interactions between the emitted ray and the medium inside the vessel and its walls, and disregarding the interactions between the ray and the detector. We can use this case as a base to understand the interactions that occur in other cases.
+
+The Case III figure depicts the evolution of the photon count in absence of the attenuation due to the medium found inside the vessel and the vessel's wall. Since we use the same set of positions in all cases, :math:`\omega(\alpha)` and :math:`\omega(\theta)` remain the same for each given position of the tracer particle. The length of the path traveled by the photon inside the detector should also be the same given the large number of Monte Carlo iterations. As seen on the Case III figure, when the particle is aligned with the axis of symmetry of the detector, the photon count reaches a maximum. At that position, the evolution of the product :math:`\omega(\alpha) \cdot \omega(\theta)` seen on the Case II figure also reaches a maximum. And the distance :math:`d(\alpha,\theta)` reaches a local maximum at that position. On the case III figure, we notice that the inflection points at :math:`y \approx -5.5` cm and at :math:`y \approx -3.7` cm (not too far from the edge of the detector's face), seen on the Case II figure, are not present anymore. This means that when :math:`y \in ]-10, -3.8[` cm, when the particle sees both the face and the lateral sides of the detector and as the particle approaches the detector's face, the distance :math:`d(\alpha,\theta)` increases making the count increase. And when :math:`y \in ]-3.8, -1.5[` cm the distance :math:`d(\alpha,\theta)` decreases in such way that it counters the rapid increase in weighting factors giving the evolution of the photon count a more parabolic shape. Finally, between :math:`y \in ]-1.5, 0]` cm, :math:`d(\alpha,\theta)` increases until reaching a local maximum.
```

Comment:
The length of the path traveled by the photon inside the detector should also be the same given the large number of Monte Carlo iterations.

I don't get this sentence.

Reply:
You're right, it's wrong. It should rather be "given that the same seed number is used."
I'll change that.